model/llm/prompt/parse_test.go

```diff
@@ -67,7 +82,9 @@ func TestParseResponse(t *testing.T) {
 			Response: "Some text...\n\n```\n" + code + "\n```\n\nSome more text...",
 
 			ExpectedAssessment: metrics.Assessments{
+				metrics.AssessmentKeyResponseNotEmpty: 1,
 				metrics.AssessmentKeyResponseNoExcess: 0,
```

Comment:
SInce there is "Some more text", excess should be 1?

Reply:
No - there is additional text. So no points for `NoExcess`. It's correct the way it is.

Reply:
AH, right! Negative!